Java/Java.sublime-syntax

```diff
@@ -227,7 +338,15 @@ contexts:
                 - match: "}"
                   pop: true
                 - include: class-body
+
   keywords:
+    - match: '::'
```

Comment:
From reading about this it seemed as though most authors considered it more of an operator, in that it transforms something, rather than a basic accessor. So let's go with `keyword`.